packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -103,6 +103,11 @@ export class FluidFetchReader extends ReadDocumentStorageServiceBase implements
         }
         throw new Error(`Can't find blob ${sha}`);
     }
+
+    public async readString(sha: string): Promise<string> {
```

Comment:
Would be good to add @inheritDoc TSDoc comments on each implementation of readString.